### PR TITLE
Document imageLabels field in BuildConfigs

### PR DESCRIPTION
**Pull request by mmilata**
Documentation for openshift/origin#11209.
Trello card: https://trello.com/c/HB9IwGj5/925-5-add-arbitrary-labels-to-docker-images

### CODE REVIEW

File: install_config/build_defaults_overrides.adoc [hunk 1]
```diff
@@ -30,16 +30,17 @@ new settings.
...
+values that have been configured for a specific build. However, if those values
+are not present on the build definition, they are set to the default value.
 
 The `*BuildOverrides*` admission control plug-in allows administrators to
 override a setting in a build, regardless of the value stored in the build. It
 currently supports overriding the `*forcePull*` flag on a build strategy to
 enforce always refreshing the local image during a build by pulling the image
 from the registry. This ensures that a user can only build with an image that
-they are allowed to pull.
+they are allowed to pull. The plugin can also be configured to apply a set of
```
Comment (bfallonf) [inline install_config/build_defaults_overrides.adoc]: s/plugin/plug-in

**Comment by bfallonf** [comment]
@mmilata One typo-level fix then I'll merge. Thanks!

**Comment by bparees** [comment]
lgtm, pending @bfallonf's nit.

**Comment by mmilata** [comment]
@bfallonf, should be fixed now.

**Comment by bfallonf** [comment]
Great! Thanks @mmilata . I'll merge away.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../dev_guide/builds.adoc#dev-guide-builds[Builds]
|Added example for custom build image label.
%
|xref:../install_config/build_defaults_overrides.adoc#install-config-build-defaults-overrides[Configuring Global Build Defaults and Overrides]
|Added image label example to build file examples.
%

**Pull request merged**